pkg/upgrade/upgrade.go

```diff
@@ -47,6 +47,13 @@ const (
 	contextTimeout    = 5 * time.Minute
 	backoffInterval   = 5 * time.Second
 	backoffMaxRetries = 5
+
+	// FlagCatalogNamespace is the name of the catalog namespace flag.
+	FlagCatalogNamespace = "catalog-namespace"
+	// FlagSkipEnvDetection is the name of the skip env detection flag.
+	FlagSkipEnvDetection = "skip-env-detection"
+	// FlagSkipOLM is the name of the skip OLM flag.
+	FlagSkipOLM = "skip-olm"
```

Comment:
These flag constants are declared multiple times, would be nice if we can declare in one place and re-use in all 3 commands, wdyt?

Reply:
I considered this originally but then I thought even though they have the same string representation, they may have a slightly different meaning and I did not want to couple them together across commands.
If they were common constants, changing them seemed prone to errors - i.e. we may want to change one command but inadvertently change the other commands as well.